data_structures/red_black_tree.ts

```diff
@@ -101,8 +101,6 @@ const {
  * @typeparam T The type of the values being stored in the tree.
  */
 export class RedBlackTree<T> extends BinarySearchTree<T> {
-  declare protected root: RedBlackNode<T> | null;
```

Comment:
Is this breaking?

Reply:
Technically yes, but it's leftover from https://github.com/denoland/deno_std/pull/4827 and breakingness of this change is covered by #4827 because this one and #4827 belong to the same release. So I avoid tag this commit as 'BREAKING'